### PR TITLE
1010D Final Content Updates for User Testing (Part 1 of 2)

### PR DESCRIPTION
**Pull request by michaelclement**
## Summary

PART 1 of 2

This ticket is part of a larger effort to update content in line with latest CAIA feedback. Changes include a multitude of wording changes and updates to schema structures. Also removed unused helpers/components and restructured file uploads substantially with custom wording.

**Detailed Change List**:

_Applicant information_
- /applicant-information/X/ssn -> Remove subtitle text and add hint to VA file number
- /applicant-infromation/X/pre-address
    - Add subtitle that reads: “We’ll send any important information about your application to this address. [Do you/does applicant] have the same address as another person listed in this application?”
    - Update question prompt to read “Mailing address”
    - Add sponsor information into dropdown
    - Add a “No, [applicant] has a different address”
    - For all available addresses, add the word “Yes, “ to the front of the select option text
    - Show this page for ALL applicants, not just those after first
- /Applicant-information/X/address -> Make sure the subtitle uses proper articles when addressing user e.g., “your” or “jimmy’s” in the subtitle
- /Applicant-information/X/email-phone
    - swap order of fields
    - Add *required to phone
    - Add subtitle: “This information helps us contact [you] faster if we need to follow up with [you] about [your] application”
- /Applicant-information/X/relationship (LIVING)
    - Update question prompt to read: “What’s [the applicant’s] relationship to sponsor?
    - Update spouse question to read: “Spouse or partner from a legal union (including a civil union or common-law marriage)
    - Update child question to read: “Child (including adopted children or step children)
    - Drop divorced question
    - Update ‘other’ question to read: “Other relationship”
- /applicant-information/X/marriage-dates (LIVING)
    - Change header to read “[Applicant] date of marriage to sponsor”
    - Add subtitle: “If [you don’t /the applicant doesn’t] know the exact date, enter [your/their] best guess.”
- /applicant-information/X/child-info (LIVING)
    - Remove (continued)
    - Update bio question wording to: “[] the sponsor’s biological child”
    - Update step question to “[] the sponsor’s stepchild”
    - Update adopted to “[] the sponsor’s adopted child”
    - Remove ‘other’ option
- /applicant-information/X/school-age (LIVING)
    - Change question text to: “Which of these best describes [you/applicant]?”
    - Change question one text to: “[] is between the ages of 18 and 23 years old and is enrolled as a student in a high school, college, or vocational school.”
    - Add a second question with text: “[] is between the ages of 18 and 23 years old and plans to enroll as a student in a high school, college, or vocational school.
    - Remove “other” option 
- /applicant-information/X/relationship (DECEASED)
    - Change question text to: “What’s [the applicant’s] relationship to the sponsor?
    - Question 1: “Surviving spouse or partner from a legal union (including a civil union or common-law marriage)
    - Question 2:  “Surviving child (including adopted children or step children)”
    - Question 3: “Other relationship”
- /applicant-information/X/marriage-details (DECEASED)
    - Change question text to: “Which of these best describes [you/the applicant]’s marriage to [their/your] sponsor?
    - Question one: change “did not” to “didn’t”
    - Change legal separation question to read: “ I was legally separated from the sponsor before their death”
    - Move remarried after 55 to last position
    - Change remarried after 55 to read: “their death” instead of “Sponsor’s death”
- /applicant-information/X/remarriage-viable (DECEASED)
    - Change title to : “[] remarriage details”
    - Change question to: “Did [your/applicant’s]  remarriage legally end (including divorce or an annulment)?
    - Change options to simply be “Yes” and “No”
- /applicant-information/X/remarried-dates (DECEASED)
    - Update first date prompt to read: “Date of marriage to [your/their] sponsor”
    - Update second date prompt to read: “Date of marriage to [your/their] current spouse”
- /applicant-information/X/remarried-dates (DECEASED)
    - See previous text updates
    - Update third prompt to read: “Date of legal separation from current spouse”
- /applicant-information/X/child-info (DECEASED)
    - Update question prompts to read:
    - “[] is the sponsor’s surviving [biological/step/adopted]child”
- /applicant-information/X/medicare-status
    - Update question to be: “Is [applicant] enrolled in Medicare Parts A & B?”
    - Add third option that reads: “No, [] is eligible for Medicare but has not been signed up for it yet”
    - Remove “other”
    - Update logic now that “part A” and “part B” will be a boolean instead of a string (see proceeding page removal)
- /applicant-information/X/medicare-status-continued -> Remove page 
- Add Medicare part D question page
- /applicant-information/X/ohi
    - change question to read: “does [] have other health insurance (other than Medicare)?
    - Change answers to be simple Yes/No only

## Related issue(s)

- Part 2: https://github.com/department-of-veterans-affairs/vets-website/pull/28835

## Testing done

- Manual testing
- Adjusted existing unit tests to match content updates

## What areas of the site does it impact?

This form only

### Quality Assurance & Testing

- [X] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [X] Browser console contains no warnings or errors.
- [X] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

NA

### CODE REVIEW

File: src/applications/ivc-champva/10-10D/config/form.js [hunk 1]
```diff
@@ -39,7 +39,6 @@ import getNameKeyForSignature from '../helpers/signatureKeyName';
 import {
   getAgeInYears,
   isInRange,
-  getParts,
```
Comment (michaelclement) [inline src/applications/ivc-champva/10-10D/config/form.js:42]: No longer used, so removing import and (further down) deleting this function

File: src/applications/ivc-champva/10-10D/pages/ApplicantMedicareStatusContinuedPage.jsx [hunk 2]
```diff
@@ -1,190 +1,87 @@
-import React, { useEffect, useState } from 'react';
```
Comment (michaelclement) [inline src/applications/ivc-champva/10-10D/pages/ApplicantMedicareStatusContinuedPage.jsx:1]: Simplify this page to just be radio buttons with dynamic label text

File: src/applications/ivc-champva/10-10D/pages/ApplicantSponsorMarriageDetailsPage.jsx [hunk 3]
```diff
@@ -188,80 +182,116 @@ export function depends18f6(formData, index) {
   );
 }
 
+function marriageTitle(text, subtitle) {
```
Comment (michaelclement) [inline src/applications/ivc-champva/10-10D/pages/ApplicantSponsorMarriageDetailsPage.jsx:185]: Broke this and the proceeding schemas out to allow for cleaner re-use

**Review by balexandr — APPROVED** [review]
lgtm

**Review by cloudmagic80 — APPROVED** [review]

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Pull request merged**